packages/eslint-plugin/src/configs/import.ts

```diff
@@ -0,0 +1,69 @@
+export = {
+  extends: ["plugin:import/recommended"],
+  plugins: ["import"],
+  rules: {
+    "import/no-unresolved": [
```

Comment:
I think this svg rule can be removed since we are not using `svg-module` anymore.

Reply:
Removed in [29ee0ab](https://github.com/WordPress/openverse/pull/2973/commits/29ee0abbd725a1053ef39a7018dda1c26064b972)